apps/datahub/src/app/record/header-record/header-record.component.ts

```diff
@@ -100,7 +100,7 @@ export class HeaderRecordComponent {
     private searchService: SearchService
   ) {}
 
-  reuseLinkUrl$ = this.facade.otherLinks$.pipe(
+  reuseLinkUrl$ = this.facade.allLinks$.pipe(
```

Comment:
Should we also update the metadata model to make explicit the fact that reuse records can have other kinds of links as well?